fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/LuceneIndexMaintenanceTest.java

```diff
@@ -576,14 +576,21 @@ void chaosMergeAndUpdateTest() throws InterruptedException, IOException {
 
             // busy merge
             for (int i = 0; i < countReps; i++) {
-                explicitMergeIndex(index, contextProps, schemaSetup);
+                try {
+                    explicitMergeIndex(index, contextProps, schemaSetup);
+                } catch (Exception e) {
+                    LOGGER.debug("couldn't merge at iteration{}", i);
```

Comment:
This will silently ignore all failures of the merger thread - should we ensure that at least some of them succeeded?